config.dev.yaml

```diff
@@ -1,3 +1,13 @@
 telegram:
-#  session_storage: "tguser.session"
-  session_storage: "tgbot.session"
\ No newline at end of file
+  #  session_storage: "tguser.session"
+  session_storage: "tgbot.session"
+  api_id: ""
```

Comment:
If it's an example - please put into an example file.

Right now this will not error out on config parsing if you won't provide those keys, but will panic somewhere later, which is a much worse UX.

This config actually gets merged with `config.dev.local.yaml`, so I don't think it's appropriate to have empty values for some keys.

I suggest making a `config.dev.local.example.yaml` for that and removing keys that _are actually not set_ from the real config.